caa2021_packages_abstract.md

```diff
@@ -1,33 +1,38 @@
 ---
 
 Standard session proposal for CAA2021, Cyprus  
 Convened on behalf of the *Scientific Scripting Languages in Archaeology* special interest group (SIG-SSLA)
 
 ## Abstract
 
-The increasing use of scientific programming languages (e.g. R or Python) is transforming the practice of computational archaeology.
-This "tool-driven revolution" [@Schmidt2020] promises to greatly improve the accessibility, power, and reproducibility of quantitative analyses.
-But a tool-driven revolution needs tools.
+The increasing use of scientific programming languages (e.g. R or Python) is changing the way computational analysis are conducted in archaeology.
+This "tool-driven revolution" [@Schmidt2020] promises to greatly improve the accessibility, power, and reproducibility of quantitative analyses and thereby to permanently influence the field as a whole [@Kristiansen2014].
+
+But a tool-driven revolution dies without versatile and sturdy tools.
 As inveterate methodological borrowers, we can frequently rely on implementations in other fields, but the adoption of scripted analysis also reiterates the long-established need for methods designed specifically for archaeological data and archaeological problems [@Kintigh1987;@Aldenderfer1998].
-As a result, recent years have seen a proliferation in packages developed by and for archaeologists (<<http://open-archaeo.info/>>).
+Recent years have seen a proliferation in packages developed by and for archaeologists (e.g. <<http://open-archaeo.info/>>).
 An increasing number of computational archaeologists therefore find themselves not only in the role of  analyst, but also that of a 'research software engineer' [@Baxter2012]; not just using tools, but making them.
 The distinct set of skills and practices this role demands has not yet been widely discussed within the field, but establishing what constitutes 'good' software engineering in archaeology is vital if we are to ensure that our new tools do what they say they do, work together, can be maintained over the long term, and are accessible to the broadest possible community of archaeological practitioners.
 
 This session, organised on behalf of the 'scientific scripting languages' special interest group, will survey the state of the art in archaeological packages for R, Python, and other scientific programming languages.
-We invite technical or theoretical papers on new packages;
-significant updates to existing packages;
-general concepts in package development as applied to archaeology (e.g. user interface design, unit testing, continuous integration, software peer review);
-critical reviews of software support for specific domains of analysis;
-or discussions of future priorities for package development in archaeology.
-The session is aimed at both developers and users of scientific programming packages in archaeology, as well as prospective users or developers.
+We invite technical or theoretical papers on
+
+- new packages
```

Comment:
can we put new packages and updates to the back of the list? I think that this is the least important!?